tools/travis-linux.sh

```diff
@@ -5,11 +5,21 @@ set -ev
 CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "$CURRENT_PATH/travis-linux-common.sh"
 
+build_without_features() {
+  echo '> Building foxbox without default features.'
+  cargo build --no-default-features
+}
 
-build() {
+build_default() {
+    echo '> Building foxbox with default features.'
     cargo build
 }
 
+build() {
+  build_without_features
```

Comment:
Nit: The other internal functions are prefixed by '_'. We could either rename `build_*` or change the implicit rule. I don't have a strong opinion on this.